views/partials/_contact-panel.njk

```diff
@@ -30,6 +30,12 @@
         <a class="govuk-link app-contact-panel__link" href="/community/propose-a-content-change-using-github/">how to propose changes in GitHub</a>.
       </p>
   {% endif %}
+  {% if section === 'Components' or section === 'Patterns' %}
+    <p class="govuk-body">
+      Tell us if your service uses this {{ section.slice(0, -1) | lower }} on our <a class="govuk-link app-contact-panel__link" target="_blank" href="https://surveys.publishing.service.gov.uk/s/MPR0MV/">usage survey (opens in a new tab)</a>. This will help us improve this {{ section.slice(0, -1) | lower }} to better meet the needs of the services that use it.
+    </p>
+  {% elseif section === 'Styles' %}
+  {% endif %}
```

Comment:
```suggestion    
        {% if section === 'Components' or section === 'Patterns' %}
    <h3 class="govuk-heading-m govuk-!-padding-top-7" id="tell-us-if-your-service-uses">Tell us if your service uses this {{ section.slice(0, -1) | lower }}</h2>
    <p class="govuk-body"><a class="govuk-link app-contact-panel__link" target="_blank" href="https://surveys.publishing.service.gov.uk/s/MPR0MV/">Take part in our usage survey (opens in a new tab)</a> to help us improve this {{ section.slice(0, -1) | lower }} to better meet the needs of the services that use it.
    </p>
  {% elseif section === 'Styles' %}
  {% endif %}
```

Reply:
Looks good to me, thanks for the suggestion!

Reply:
I was wrong, linting error